auditeval/test.go

```diff
@@ -302,6 +302,16 @@ func compareOp(tCompareOp string, flagVal string, tCompareValue string) (bool, s
 		s := splitAndRemoveLastSeparator(flagVal, defaultArraySeparator)
 		target := splitAndRemoveLastSeparator(tCompareValue, defaultArraySeparator)
 		testResult = allElementsValid(s, target)
+
+	case "bitmask":
+		expectedResultPattern = "bitmask '%s' AND '%s'"
+		requested, err := strconv.ParseInt(flagVal, 8, 64)
```

Comment:
```suggestion
		requested, err := strconv.ParseInt(flagVal, 8, 64)
		if err != nil {
		    ...complete me...
		}
```

You need to test err from both of the calls to `strconv.parseInt()`

Reply:
add the same check as I did for the other `strconv.parseInt()` function because we just output all the variables anyway